presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -479,9 +479,12 @@ else if (!partitionColumnNames.contains(name)) {
         table.setPartitionKeys(partitionColumns);
         table.setSd(sd);
 
-        PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, tableOwner, PrincipalType.USER, true);
+        PrivilegeGrantInfo selectPrivilege = new PrivilegeGrantInfo("select", 0, tableOwner, PrincipalType.USER, true);
```

Comment:
ah I see, I didn't look for that detail. Our customers experienced the issue and asked for a fix. Will let you know if I find out more about it.